spec/models/miq_product_feature_spec.rb

```diff
@@ -112,20 +112,18 @@ def traverse_product_features(product_feature, &block)
     it "returns only visible features" do
       hierarchical_features
       expect(MiqProductFeature).not_to receive(:sort_children)
-      expect(MiqProductFeature.feature_children("miq_report_widget_admin", false)).to match_array(
-        %w(widget_copy widget_edit))
+      expect(MiqProductFeature.feature_children("miq_report_widget_admin", false)).to match_array(%w(widget_copy widget_edit))
     end
 
     it "returns direct children only" do
       hierarchical_features
-      expect(MiqProductFeature.feature_children("miq_report_widget_editor")).to eq(
-        %w(miq_report_widget_admin))
+      expect(MiqProductFeature.feature_children("miq_report_widget_editor")).to match_array(%w(miq_report_widget_admin))
```

Comment:
For consistency...the other three tests are all physically structured the same way